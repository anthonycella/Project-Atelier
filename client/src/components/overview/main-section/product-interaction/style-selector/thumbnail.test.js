import renderer from 'react-test-renderer';
import Thumbnail from './thumbnail.jsx';

it('Thumbnail renders to the page', () => {
  const size = '5px';
  const imgUrl = 'https://static.wikia.nocookie.net/lotr/images/1/1a/FotR_-_Elijah_Wood_as_Frodo.png/revision/latest?cb=20130313174543';
  const component = renderer.create(
    <Thumbnail imgUrl={imgUrl} size={size} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
