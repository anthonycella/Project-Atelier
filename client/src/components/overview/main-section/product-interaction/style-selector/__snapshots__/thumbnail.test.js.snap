// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbnail renders to the page 1`] = `
<div
  className="sc-bczRLJ jprvzM"
  size="5px"
/>
`;
